packages/assets-listed/src/index.ts

```diff
@@ -0,0 +1,22 @@
+export const ASSETS_LISTED = [
+  {
+    assetId:
+      '0x0000000000000000000000000000000000000000000000000000000000000000',
+    name: 'Ethereum',
+    symbol: 'ETH',
+    imageUrl: `assets/eth.svg`,
+  },
+  {
+    assetId:
+      '0xe90f62d5216f58ffaf19e6eba07f944cc2cd36f9800b89419d52b6927f53976701',
+    name: 'Dai',
+    symbol: 'DAI',
+    imageUrl: `assets/dai.svg`,
+  },
+  {
+    assetId: '0x2260fac5e5542a773aa44fbcfedf7c193bc2c599',
+    name: 'Bitcoin',
+    symbol: 'BTC',
+    imageUrl: `assets/btc.svg`,
+  },
+];
```

Comment:
We can remove DAI and Bitcoin for now as we don't support this assets, keep just ETH for now.